README.md

```diff
@@ -44,6 +45,14 @@ See [**Contributing.md**](./CONTRIBUTING.md) to see how to format your pull requ
 - **Issues**: [adobe](https://github.com/search?q=user%3Aadobe+type:issue&state=open&type=Issues), [magento](https://github.com/search?q=user%3Amagento+type:issue&state=open&type=Issues)
 - **Notes**: See more on [official blog](https://community.magento.com/t5/Magento-DevBlog/Squashtoberfest-2019/ba-p/143694) and their [Squashtoberfest repo](https://opensource.adobe.com/squashtoberfest/)
 
+#### **AntiGravity OverUnity Company Inc.**
```

Comment:
Please also add to the "Least Involvement to Most Involvement" section of the list.